sys/include/net/netstats.h

```diff
@@ -41,6 +50,16 @@ typedef struct {
     uint32_t rx_bytes;          /**< received bytes */
 } netstats_t;
 
+/**
+ * @brief   Get a string description for @ref netstats_module_t, for debugging
```

Comment:
Apparently I remembered that one wrong (or got confused by vim's syntax highlighting), but the discussion is obsolete anyway - the function is gone. :)